### ISSUE TITLE
Included files have wrong timestamps

### ISSUE THREAD

**Issue by Schroedingers-Cat** [chocolatey/choco#1423]
### What You Are Seeing?
Files in packages included via nuspec's <file> tag have the timestamp from packaging.

### What is Expected?
Files in packages included via nuspec's <file> tag should have the timestamp of the original files as everything else can cause confusion about versions, authenticity etc. .

### How Did You Get This To Happen? (Steps to Reproduce)
Include some files via nuspec's <file> tag, build the package, open it with an archive browser like 7zip and browse to the included files. Also installing the package will make the extracted files have the timestamp from the archive and not the original file's timestamp.

First reported overe [here](https://groups.google.com/forum/#!topic/chocolatey/ZoKz9mM_CFQ).

**Comment by ferventcoder** [comment]
Thank you. We will triage this before the next release, but depending on the implementation details, it may or may not be possible as easily as we are hoping for.

**Comment by Schroedingers-Cat** [comment]
Has there been any progress on this?

**Comment by ferventcoder** [comment]
@Schroedingers-Cat no progress from our side yet. I apologize if my earlier statement was not very clear. 

A good workaround for now would be to pack things into a zip file and place the zip file in the Chocolatey package.

**Comment by ferventcoder** [comment]
Also with really good ways to verify authenticity, and versioning being handled at the package level, this feels to be lower priority. Does that sound appropriate?

**Comment by ViceIce** [comment]
Additionally it would be nice to preserve the timestamps when unpacking files.

**Comment by Schroedingers-Cat** [comment]
True, that's why my current workaround is to zip them before packaging. Makes things a bit more complicated, though. So would be cool to have this fixed directly in chocolatey.

**Comment by pauby** [comment]
Thanks for raising this issue. After reviewing it, it's not something we expect to prioritise in the foreseeable future. We’re closing it to keep our backlog focused, but we’re open to revisiting it in the future if priorities shift or new interest arises. You are welcome to comment here or open a new issue if you'd like to discuss it further down the line.
